blivet/devicelibs/dasd.py

```diff
@@ -114,6 +114,10 @@ class dasd_info(ctypes.Structure):
 
 def is_ldl_dasd(device):
     """Determine whether or not a DASD is LDL formatted."""
+    statusfile = "/sys/block/%s/device/status" % (device,)
+    if not os.path.isfile(statusfile):
+        return False
```

Comment:
I did consider that, but it's honestly cleaner and easier if it's done here.

Anaconda is passing a list of device names, not objects, so at best it could only really see if each of those names was in storage.devicetree.dasd before calling is_ldl_dasd. That might be a "meh, fine" level of ugliness, except the line that is collecting the list of LDL DASDs is:

`ldldasds = [d for d in self.selected_disks if is_ldl_dasd(d)]`